8Knot/app.py

```diff
@@ -93,3 +93,32 @@
 
 if os.getenv("8KNOT_DEBUG", "False") == "True":
     app.enable_dev_tools(dev_tools_ui=True, dev_tools_hot_reload=False)
+
```

Comment:
Create a file called _bots.py that's neighbors with _celery.py and move this code inside of some function like "get_bots_list()", then import the function `import _bots as bots` and run `bots_list = bots.get_bots_list()`

Reply:
I wish I would have thought about this before changing all of the files but this solution would have the same query running for each visualization vs once and then the list is accessed

Reply:
I should have been more specific- call `bots_list = bots.get_bots_list()` only once, creating the object 'bots_list' in app.py, and then use that list like you were before by grabbing it from app.py at runtime.

Reply:
Oof I just spent like an hour doing all of those changes